routes/problems_id.js

```diff
@@ -26,11 +27,24 @@ async function getProblem(id) {
   return problems[0];
 }
 
+/**
+ * Helper function that calculates code coverage.
+ * Multiplies the timeout given by CODE_COVERAGE_TIMEOUT_MULTIPLIER
+ * to account for instrumented code taking longer to run
+ * and returns it as a datastore double.
+ **/
+async function calculateCodeCoverage(code, timeout) {
+  const coverageTimeout = timeout * CODE_COVERAGE_TIMEOUT_MULTIPLIER;
+  const codeCoverage = await sandbox.calculateCodeCoverage(code);
```

Comment:
it doesn't look like the timeout is used, did u mean to pass it as a param to `calculateCodeCoverage`

Reply:
Thanks for the catch!
Fixed it :)